core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GoogleServicesDAO.scala

```diff
@@ -231,6 +231,13 @@ abstract class GoogleServicesDAO(groupsPrefix: String) extends ErrorReportable {
   def getFolderId(folderName: String): Future[Option[String]]
 
   def testBillingAccountAccess(billingAccount: RawlsBillingAccountName, userInfo: UserInfo): Future[Boolean]
+
+  /**
+    * Returns location of a regional bucket. If the bucket's location type is `multi-region`, it returns None
```

Comment:
Is that right? How do we know how to clone the region requirements for multi-regional buckets?

Reply:
Can you explain this more? We are only supporting creating/cloning workspaces (outside us) in single region only no at the moment?